### PR TITLE
Add description of line and dot marks' position

### PR DESCRIPTION
**Pull request by bobbytung**
Just before 3.1.1, to solve issue #192 and #176

***
<a href="https://pr-preview.s3.amazonaws.com/bobbytung/clreq/pull/200.html" title="Last updated on Jan 16, 2019, 11:41 AM UTC (ceb67ae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/200/1bb20c0...bobbytung:ceb67ae.html" title="Last updated on Jan 16, 2019, 11:41 AM UTC (ceb67ae)">Diff</a>

### CODE REVIEW

**Comment by xfq** [comment]
@r12a, could you have a look at this PR?

**Comment by r12a** [comment]
Yes, i actually started looking at it yesterday.  I think that the information that is added in the PR is already available at https://w3c.github.io/clreq/#indication_punctuation_marks, bullet points 8 and 9 (aside: we really should add some more link anchors to places like this in the doc).  And in fact most of it is mentioned again near the end of https://w3c.github.io/clreq/#glyphs_sizes_and_positions_in_character_faces_of_punctuation_marks

What's not mentioned in any of those places or the new text is:
1. When two different proper nouns appear side-by-side, the underline should be broken between the two. (It may be elsewhere, and i missed it.)
2. If a line of Chinese text contains some text in Japanese or another language, and there is a line associated with that text, the position of the line should follow the Chinese conventions.

The latter was information that CSS folks were looking for, so i think it should be covered in clreq.

So, in summary, my conclusion was that, unless i'm missing something (quite possible), what the PR adds is redundant, and that what needs to be added is not there.

**Comment by xfq** [comment]
Yeah, after taking a closer look at this PR, I believe it hasn't solved issue #176 and #192.

(Although after some editing, it could be a fix for #113 ;)

**Comment by xfq** [comment]
Since we have similar text in https://w3c.github.io/clreq/#glyphs_sizes_and_positions_in_character_faces_of_punctuation_marks , I will close this PR.
